cl/api/utils.py

```diff
@@ -604,24 +606,48 @@ def check_webhook_failure_count_and_notify(
         1: True,  # Send first webhook failing notification
         2: False,
         3: False,
-        4: True,  # Send second webhook failing notification
-        5: False,
-        6: True,  # Send third webhook failing notification
+        4: False,
+        5: True,  # Send second webhook failing notification
+        6: False,
         7: True,  # Send webhook disabled notification
     }
-
-    current_try_counter = webhook_event.retry_counter
-    disabled = False
     webhook = webhook_event.webhook
-    if current_try_counter >= WEBHOOK_MAX_RETRY_COUNTER and webhook.enabled:
-        webhook.enabled = False
-        # Don't send notification email via signal in cl.users.signals
-        webhook.save(update_fields=["enabled"])
-        disabled = True
+    if not webhook.enabled or webhook_event.debug:
+        return
 
+    webhook.failure_count = F("failure_count") + 1
+    update_fields = ["failure_count"]
+
+    current_try_counter = webhook_event.retry_counter
     notify = notify_on[current_try_counter]
     if notify:
-        notify_failing_webhook.delay(webhook_event.webhook.pk, disabled)
+        disabled = False
```

Comment:
I'd ditch this variable and just use `webhook.enabled` instead, since it always has this same meaning and value.

Reply:
Yeah, it's possible just use `webhook.enabled`, changed.